src/poggit/ci/builder/ProjectBuilder.php

```diff
@@ -665,9 +661,9 @@ protected function checkPhp(BuildResult $result, string $iteratedFile, string $c
                     $namespaces[] = $currentNamespace = $buildingNamespace;
                     unset($buildingNamespace);
                 }
-            } elseif($tokenId === T_CLOSE_TAG) {
```

Comment:
Prefer separating parser conditions and config conditions into two nested if statements. This would reduce the risk of parsing inconsistency.

Reply:
Sorted